samples/forseti/forseti_conf.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##################################################################
+# gsuite
+##################################################################
+# The fully qualified path and filename to the key file to access GSuite.
+# REQUIRED if inventory groups and running locally.
+gsuite_service_account_key_file: /usr/local/google/home/henryc/git/forseti-security/super_admin_key.json
+
+# An email address of a super-admin in the GSuite domain.
+# REQUIRED if inventory groups.
+domain_super_admin_email: henry@henrychang.mygbiz.com
```

Comment:
s/domain_super_admin_email/gsuite_super_admin_email/

This will also require doc changes.